src/main/java/org/spdx/sbom/gradle/SpdxSbomPlugin.java

```diff
@@ -130,12 +129,21 @@ private void createTaskForTarget(
                   t.usesService(knownLicenseServiceProvider);
 
                   List<String> configurationNames = target.getConfigurations().get();
+                  var rootComponentsProperty =
+                      project.getObjects().listProperty(ResolvedComponentResult.class);
                   for (var configurationName : configurationNames) {
                     Provider<Set<ResolvedArtifactResult>> artifacts =
                         project
                             .getConfigurations()
                             .getByName(configurationName)
                             .getIncoming()
+                            .artifactView(
+                                viewConfiguration ->
+                                    viewConfiguration.attributes(
+                                        attributes ->
+                                            attributes.attribute(
+                                                Attribute.of("artifactType", String.class),
+                                                "android-aar-or-jar")))
```

Comment:
This turned out to be the solution for resolving normal artifacts. I was inspired by https://github.com/gradle/gradle/issues/20779.

Reply:
`Attribute.of("artifactType", String.class)` comes from [here](https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:build-system/gradle-core/src/main/java/com/android/build/gradle/internal/publishing/AndroidArtifacts.java;l=34?q=AndroidArtifacts.java)

`"android-aar-or-jar"` comes from [here](https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:build-system/gradle-core/src/main/java/com/android/build/gradle/internal/publishing/AndroidArtifacts.java;l=575?q=AndroidArtifacts.java)